tests/apps/test_graph_utils.py

```diff
@@ -186,3 +186,91 @@ def test_invalid_subgraphs(self, graph, dim):
         subgraphs = itertools.combinations(range(dim), int(dim / 2))
 
         assert not all([utils.is_subgraph(list(s), graph) for s in subgraphs])
+
+
+@pytest.mark.parametrize("dim", range(2, 10))
+class TestIsClique:
+    """Tests for the function `strawberryfields.apps.graph.utils.is_clique` """
+
+    def test_no_false_negatives(self, dim):
+        """ Tests that cliques are labelled as such"""
+        g = nx.complete_graph(dim)
+        assert utils.is_clique(g)
+
+    def test_no_false_positives(self, dim):
+        """ Tests that non-cliques are labelled as such"""
+        g = nx.empty_graph(dim)
+        assert not utils.is_clique(g)
+
+
+@pytest.mark.parametrize("dim", range(2, 10))
+class TestC0:
+    """ Tests function c_0 that generates the set of nodes connected to all nodes in a clique"""
```

Comment:
I notice in a handful of places there's an extra space after the leading quotes (`""" `), is this intentional?

Reply:
I saw this but didn't comment on it, thinking it might be nitpicking :laughing: 
But since @jswinarton has noticed it as well, in the rest of the SF documentation (barring some backends) we tend to stick to no-space and no-new-line (i.e., `"""Test function...`)